frontend/src/core/comments/components/Comment.js

```diff
@@ -59,7 +71,27 @@ export default function Comment(props: Props) {
                     date={ new Date(comment.dateIso) }
                     title={ `${comment.createdAt} UTC` }
                 />
+                { canPin ?
+                    <Localized
+                        id="comments-Comment--pin-button"
+                        attrs={{ title: true }}
+                        vars={{
+                            pinnedTitle: !comment.pinned ? 'Pin comment' : 'Unpin comment',
+                            pinButton: !comment.pinned ? 'PIN' : 'UNPIN',
```

Comment:
You could also change the ID for the button.

Re UPPERCASE, using CSS isn't good for l12y, I think.

Reply:
> Re UPPERCASE, using CSS isn't good for l12y, I think.

Interesting. I suggested CSS because using uppercase here is a stylistic choice that might not be carried over to a future redesign.

I'm curious to learn more about the l12y concerns of using CSS to change case. Is this one of them? https://stackoverflow.com/questions/28783259/how-do-i-make-text-transformuppercase-work-properly-with-greek

Reply:
The https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Create_localizable_UI linked from there is more to the point. Greek might be solved by now, not sure Gaelic is. IIRC, Turkish also has letters that are "interresting" to uppercase.

And then there's the question around uppercase being the right choice for a language to begin with. Similar to the Italics comment for CJKT on that page.

Reply:
Thanks. We use `text-transform: uppercase` quite a lot in the Translate app, so I'll file a bug to change that.

Reply:
Filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1666277